features/FEATURE_BLE/ble/SecurityManager.h

```diff
@@ -21,9 +21,30 @@
 
 #include "Gap.h"
 #include "CallChainOfFunctionPointersWithContext.h"
+#include "ble/BLETypes.h"
+
+class SecurityManagerEventHandler;
+class LegacySecurityManagerEventHandler;
+
+using ble::connection_handle_t;
```

Comment:
Remove using declarations from the header file.

Reply:
removed